pkg/pillar/types/downloadertypes.go

```diff
@@ -52,16 +52,16 @@ func (config DownloaderConfig) LogModify(logBase *base.LogObject, old interface{
 		logObject.Clone().Fatalf("LogModify: Old object interface passed is not of DownloaderConfig type")
 	}
 	if oldConfig.Target != config.Target ||
-		oldConfig.DatastoreID != config.DatastoreID ||
+		oldConfig.DatastoreIDsList[0] != config.DatastoreIDsList[0] ||
```

Comment:
For these comparisons can't you use Equal on the slice? Or introduce an Equal function which compares the slice of UUIDs. Don't know if that is easier if you define a new type for the list of UUIDs or use []uuid.UUID explicitly.

Reply:
Will do.